backend/app/adapter/db/url.go

```diff
@@ -61,6 +63,44 @@ VALUES ($1, $2, $3, $4, $5);`,
 	return err
 }
 
+// UpdateURL updates a URL that exists within the URL table and returns the newly updated URL if there is no error found while
+// committing the update.
+func (u *URLSql) UpdateURL(key string, newAlias string, newOriginalURL string, expireAt *time.Time) (entity.URL, error) {
```

Comment:
```suggestion
func (u *URLSql) UpdateURL(oldAlias string, newURL entity.URL) (entity.URL, error) {
```

Reply:
Will update the interface and implementations to use the new arguments.